internal/usecase/repository/domain_repository_test.go

```diff
@@ -756,8 +756,8 @@ func (s *DomainRepositorySuite) TestList() {
 	s.mock.ExpectQuery(regexp.QuoteMeta(`SELECT count(*) FROM "domains" WHERE org_id = $1`)).
 		WithArgs(orgID).
 		WillReturnRows(sqlmock.NewRows([]string{"count"}).AddRow(1))
-	s.mock.ExpectQuery(regexp.QuoteMeta(`SELECT * FROM "domains" WHERE org_id = $1 AND "domains"."deleted_at" IS NULL LIMIT 5`)).
-		WithArgs(orgID).
+	s.mock.ExpectQuery(regexp.QuoteMeta(`SELECT * FROM "domains" WHERE org_id = $1 AND "domains"."deleted_at" IS NULL LIMIT $2`)).
+		WithArgs(orgID, int(5)).
```

Comment:
why `int(5)` instead of `5`? There is no point to cast an integer to `int` here.